docs/0.1/creating_locations.md

```diff
@@ -0,0 +1,376 @@
+
+   ```
+   root@gridd-alpha:/# export GRID_DAEMON_ENDPOINT="http://localhost:8080"
+   ```
+
+1. Set `GRID_SERVICE_ID` to the fully qualified service ID (such as
+   `01234-ABCDE::gsAA`) for the scabbard service on the circuit. This
+   environment variable replaces the `--service_id` option on the `grid`
+   command line.
+
+   ```
+   root@gridd-alpha:/# export GRID_SERVICE_ID="01234-ABCDE::gsAA"
+   ```
+
+   Tip: See [Prerequisites](#prerequisites) for the `splinter` commands that
+   display the circuit ID and four-character service ID string.
+
+### Define and Create a Location
+
+Each location requires an organization ID (the owner) and at least one agent
+with permission to create, update, and delete the location. There must also
+be an existing location schema for the location's properties.
+See [Prerequisites](#prerequisites) for more information.
+
+{:start="5"}
+
+5. Create a location definition file, in YAML format, that specifies the
+   following information:
+
+   * location namespace (GS1 for this example) that corresponds with the
+     location schema name (`gs1_location` for this example).
+   * Location ID (such as a GLN)
+   * Owner (organization ID)
+   * Set of location attributes (each with a name and value that conforms to
+     the data type)
+     <br><br>
+
+    ```yaml
+    - namespace: "GS1"
+      location_id: "0013600000011"
+      owner: "013600"
+      properties:
+        locationDescription: "test location"
+        locationType: 0
+        addressLine1: "251 Test St."
+        city: "Minneapolis"
+        stateOrRegion: "MN"
+        postalCode: "55401"
+        country: "United States"
+        latLongValue: "449778,-932650"
+        contactName: "John Tester"
+        contactEmail: "tester@email.com"
+        contactPhone: "6555555555"
+        createDate: 1602706399
+    ```
+
+    Tip: Use a YAML linter to validate the new file is formatted correctly.
+
+1. Use `docker cp` to copy the file into the `gridd-alpha` container.
+
+   ```
+   $ docker cp location.yaml gridd-alpha:/
+   ```
+
+1. Add the new location by using the `grid location create` command to specify the
+   definition in the `location.yaml` file.
+
+   ```
+   root@gridd-alpha:/# grid location create --file location.yaml
+   ```
+
+   This command creates and submits a transaction to add the location data to the
+   distributed ledger. If the transaction is successful, all other nodes in the
+   circuit can view the location data.
+
+   **Note**: This command does not display any output. Instead, check the log
+   messages on the console (or the terminal window where you started the Grid
+   Docker environment) for the success or failure of this operation.
+
+### Display Location Information
+
+{:start="8"}
+
+8. List all existing locations to verify that the new location has been added.
+
+   ```
+   root@gridd-alpha:/# grid location list
+   Location ID: 9012345000004
+   Namespace: GS1
+   Owner: 314156
```

Comment:
Looks like this is not using the updated location list output (and its an old owner ID)

Reply:
updated